ports/supermethanebros/port.json

```diff
@@ -0,0 +1,32 @@
+{
+  "version": 3,
+  "name": "supermethanebros.zip",
+  "items": [
+    "Super Methane Brothers.sh",
+    "supermethanebros/"
+  ],
+  "items_opt": [],
+  "attr": {
+    "title": "Super Methane Brothers",
+    "porter": [
+      "Slayer366"
+    ],
+    "desc": "A conversion of Super Methane Brothers for the Commodore Amiga.  The gameplay is similar to Taito's Bubble Bobble.",
+    "desc_md": null,
+    "inst": "Ready to run!",
+    "inst_md": null,
+    "genres": [
+      "action",
+      "arcade",
+      "platformer"
+    ],
+    "image": null,
+    "rtr": true,
```

Comment:
As it is RTR the licence must be include. I read on the github repo this
> IMPORTANT NOTE: This is a conversion of the Commodore Amiga game. I had been given permission by the company (Apache Software Ltd) to release this game as GPL. However - THE ORIGINAL AMIGA VERSION OF SUPER METHANE BROTHERS IS STILL A COMMERCIAL GAME IT'S LICENCE HAS NOT CHANGED.

So at least GPL licence must be present.

Reply:
Where am I supposed to get the GPL license?  Which version of the GPL license?

Reply:
Kloptops and JanTrueno aided me in finding the licenses.  Hopefully we now have what we need.